playgame.py

```diff
@@ -56,20 +56,20 @@ def ensure_path(path):
 
 def run_tournament(args):
     tournament_scores = dict.fromkeys(args.players, [])
-    pairings = list(itertools.combinations(args.players, 2))
+    pairings = list(itertools.combinations(args.players, args.per_round))
     disqualified = []
```

Comment:
What do you think about making `disqualified` a `set` instead of an array? It's still iterable, has `add` for appending, expresses that the disqualified players are a unique set, and eliminates the need to create a new `set` in each iteration.

Reply:
disqualified is now a set. Thought about doing it for pairings as well, but the method checks a pairing against disqualified, not the pairings set itself